### DISCUSSION TITLE
better class names for align and justify properties

### DISCUSSION THREAD

**Discussion by babakfp** [tailwindlabs/tailwindcss#2543] [Show and tell]
Hey. Isn't better to write these classes like this:

```css

.ai /* For */ align-items     /* Currently */ .items
.ac /* For */ align-content   /* Currently */ .content
.as /* For */ align-self      /* Currently */ .self
.ji /* For */ justify-items   /* Currently */ .justify-items
.jc /* For */ justify-content /* Currently */ .justify
.js /* For */ justify-self    /* Currently */ .justify-self

```

Why do I think it's better?
- In the first look, you can know that it's related to what property.
- If you know about one (`ai` for `align-items`), you will know about the others.
- It takes up little space.

I'm just really not comfortable with those class namings.

```html
<!-- Suggested: -->
<div class="flex ai-center jc-between"><i></i>text</div>

<!-- Currently: -->
<div class="flex items-center justify-between"><i></i>text</div>
```

**Comment by subhero24** [comment]
I agree these are among the most confusing class names in tailwindcss. I would rather have the full name like `align-items-center` than the current ones.
